docs/data/date-pickers/date-time-field/CustomDateTimeFormat.tsx

```diff
@@ -1,16 +1,16 @@
 import * as React from 'react';
 import dayjs, { Dayjs } from 'dayjs';
+import { DemoContainer } from 'docsx/src/modules/components/DemoContainer';
 import { LocalizationProvider } from '@mui/x-date-pickers';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { Unstable_DateTimeField as DateTimeField } from '@mui/x-date-pickers/DateTimeField';
-import Stack from '@mui/material/Stack';
 
 export default function CustomDateTimeFormat() {
   const [value, setValue] = React.useState<Dayjs | null>(dayjs('2022-04-07T15:30'));
 
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
-      <Stack spacing={4} sx={{ '& > *': { width: 300 } }}>
+      <DemoContainer>
```

Comment:
I may be biased, but I liked the previous version with all content visible and no labels being cut-off better 🙈 
![Screenshot 2022-12-09 at 15 01 20](https://user-images.githubusercontent.com/4941090/206708322-48213ae2-8a96-4ebd-95e4-0826f1592b76.png)

Reply:
![image](https://user-images.githubusercontent.com/3309670/206989553-1fc98fe1-2eb6-4615-99e8-37e916516df3.png)

It was not cut on my side
I will add a minWidth depending on the type of field rendered

Reply:
What browser are you using?
For some reason different width fields are rendered depending on browser used.
In Chrome I get 195px field and in Safari and Firefox—230px and no cutting off 🙈 
<img width="1146" alt="Screenshot 2022-12-12 at 09 53 38" src="https://user-images.githubusercontent.com/4941090/206991195-5e6b8544-ef46-40ae-8504-26d00e38be9b.png">

Reply:
I added new rules for single input range fields and fields with a date time format.
Not sure it cover every use case but it should be better